YAXLib/Serialization.cs

```diff
@@ -40,6 +43,8 @@ internal void Initialize()
         {
             _baseElement = null;
             _mainDocument = null;
+            _stripInvalidXmlChars = (_serializer.Options.SerializationOptions &
+                                     YAXSerializationOptions.StripInvalidXmlChars) == YAXSerializationOptions.StripInvalidXmlChars;
```

Comment:
Can't we use https://docs.microsoft.com/en-us/dotnet/api/system.enum.hasflag?view=netframework-4.0

(Haven't checked platform support)

Reply:
Indeed - overlooked 😴